microscopium/preprocess.py

```diff
@@ -677,6 +677,47 @@ def _reservoir_sampled_image(ims_iter, random_state=None):
     return sampled
 
 
+def global_threshold(ims_iter, random_state=None):
+    """Generate a global threshold for the collection of images given.
+
+    The threshold is determined by sampling the intensity of every
+    image and then computing the isodata [1]_ threshold on this sample.
+
```

Comment:
My idea is to soon deprecate the whole old style, so I'll leave it for now, for *historical* consistency. (It would be Otsu that would be replaced though since it's an image-wide threshold.)